src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -230,28 +204,21 @@ public CaseDocumentAmController(DocumentManagementService documentManagementServ
     public ResponseEntity<Object> generateHashCode(
 
         @PathVariable("documentId") UUID documentId) {
-        try {
-            StoredDocumentHalResource resource = new StoredDocumentHalResource();
-            ResponseEntity responseEntity = documentManagementService.getDocumentMetadata(documentId);
-            if (responseEntity.getStatusCode().equals(HttpStatus.OK) && null != responseEntity.getBody()) {
-                resource = (StoredDocumentHalResource) responseEntity.getBody();
-            }
-
-            ValidationService.validateInputParams(INPUT_STRING_PATTERN, documentId.toString(),
-                                                  resource.getMetadata().get("caseTypeId"), resource.getMetadata().get("jurisdictionId"));
+        StoredDocumentHalResource resource = new StoredDocumentHalResource();
+        ResponseEntity responseEntity = documentManagementService.getDocumentMetadata(documentId);
+        if (responseEntity.getStatusCode().equals(HttpStatus.OK) && null != responseEntity.getBody()) {
+            resource = (StoredDocumentHalResource) responseEntity.getBody();
+        }
 
-            HashMap<String, String> responseBody = new HashMap<>();
+        ValidationService.validateInputParams(INPUT_STRING_PATTERN, documentId.toString(),
+                                              resource.getMetadata().get("caseTypeId"), resource.getMetadata().get("jurisdictionId"));
 
-            String hashedToken = ApplicationUtils.generateHashCode(documentId.toString().concat(
-                resource.getMetadata().get("jurisdictionId")).concat(resource.getMetadata().get("caseTypeId")));
-            responseBody.put(HASHCODE, hashedToken);
+        HashMap<String, String> responseBody = new HashMap<>();
 
-            return new ResponseEntity<>(responseBody, HttpStatus.OK);
+        String hashedToken = ApplicationUtils.generateHashCode(documentId.toString().concat(
+            resource.getMetadata().get("jurisdictionId")).concat(resource.getMetadata().get("caseTypeId")));
```

Comment:
Use same string which we created in above step using string builder class.

Reply:
same as above, this is used only once so string builder is not requied.